### PR TITLE
feat(cards): #MED-268 make design of cards for each categories

### PR DESCRIPTION
**Pull request by niko-amosse**
## Describe your changes

Make the cards design for each categories (pinned, book-mark, manuals, favorites). Adapt them to home page design.

image

## Checklist tests

* Behavior and display in responsive design
* Design is same as mockup and PO requests

## Issue ticket number and link

[MED-268](https://jira.support-ent.fr/browse/MED-268)

## Checklist before requesting a review (magic string, indentation, comment/documentation...)

- [ ] I have detailed the tests to do in my feature/fix in order to prevent consequents regressions (must specify in **Checklist tests**)
- [X] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (API Doc etc...) - (must specify in **Description** for target version)
- [ ] If it is a consequent feature, I have added thorough tests.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been added to this project (must specify in **Description**)

### CODE REVIEW

File: backend/.gitignore.swp
Comment (Mattox0) [inline backend/.gitignore.swp:1]: Ce fichier n'a rien à faire là

File: frontend/src/components/list-card/ListCard.scss [hunk 1]
```diff
@@ -3,95 +3,87 @@
 .list-card {
   display: flex;
   flex-direction: column;
-  margin-bottom: 10px;
+  margin: 10px;
 
-  .manuals .favorites .util_links {
+  &.pinned_resources {
+    padding: 15px 10px;
+    background-color: white;
```
Comment (Mattox0) [inline frontend/src/components/list-card/ListCard.scss:10]: met la couleur dans _colors et fait en une variable

File: frontend/src/components/list-card/ListCard.scss [hunk 2]
```diff
@@ -3,95 +3,87 @@
...
+    border-radius: 5px;
+    box-shadow: 0px 2px 3px 0px #0000001f;
+  }
+  &.favorites {
+    padding: 15px 25px;
+    background-color: white;
+    border-radius: 5px;
```
Comment (Mattox0) [inline frontend/src/components/list-card/ListCard.scss:17]: idem

File: frontend/src/components/resource/Resource.scss [hunk 3]
```diff
@@ -1,67 +1,186 @@
...
+    svg {
+      height: 18px;
+      margin-right: 1px;
+      width: 18px;
+      cursor: pointer;
+    }
+
+    .med-link {
+      cursor: pointer;
+      color: #4d54c5;
```
Comment (Mattox0) [inline frontend/src/components/resource/Resource.scss:40]: _colors

File: frontend/src/components/resource/Resource.scss [hunk 4]
```diff
@@ -1,67 +1,186 @@
...
+    }
+
+    .med-pin {
+      cursor: pointer;
+      color: #ff656a;
```
Comment (Mattox0) [inline frontend/src/components/resource/Resource.scss:45]: _colors

File: frontend/src/components/resource/Resource.scss [hunk 5]
```diff
@@ -1,67 +1,186 @@
...
+    }
+
+    .med-star {
+      color: #ffb549;
```
Comment (Mattox0) [inline frontend/src/components/resource/Resource.scss:49]: _colors

**Review by Mattox0 — APPROVED** [review]

File: frontend/src/components/list-card/ListCard.tsx [hunk 6]
```diff
@@ -61,31 +64,22 @@ export const ListCard: React.FC<ListCardProps> = ({
     return components.length > NbComponents(windowWidth);
   };
 
+  const iconMapping = {
```
Comment (alicedraillard) [inline frontend/src/components/list-card/ListCard.tsx:67]: fais un enum

Comment (niko-amosse) [reply frontend/src/components/list-card/ListCard.tsx:67]: j'ai pas l'impression de pouvoir : la clé d'un enum ne peut pas être un const (soit nombre ou chaine de charactère) :/ Je suis obligé d'utilisé un map j'ai l'impression

**Review by alicedraillard — APPROVED** [review]

**Pull request merged**